lib/Utils/Architectures.pm

```diff
@@ -128,6 +129,17 @@ sub is_ppc64le {
     return check_var('ARCH', 'ppc64le');
 }
 
+=head2 is_ppc64
```

Comment:
unrelated change?

Reply:
As I write in the description,I excluded this test from ppc64 arch in Tumbleweed. So it is not unrelated, as I added this in `Architectures.pm` to keep the same code style here : `if (!is_sle('<=12-SP3') && !is_ppc64le && !is_ppc64)`

Reply:
As it is related, do you still think that I have to add it in a different PR or commit?

Reply:
@punkioudi ~~I don't see the changes in the PR that use `is_ppc64` :)~~ Nope, forget what I wrote :)